### PR TITLE
joshuto: 0.9.3 -> 0.9.4

### PR DESCRIPTION
**Pull request by totoroot**
###### Description of changes

Update joshuto from version 0.9.3 to 0.9.4
Add myself (@totoroot) as a maintainer

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by totoroot** [comment]
Result of `nixpkgs-review pr 182736` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>joshuto</li>
  </ul>
</details>

**Comment by totoroot** [comment]
Result of `nixpkgs-review pr 182736` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>joshuto</li>
  </ul>
</details>

File: pkgs/applications/file-managers/joshuto/default.nix [hunk 1]
```diff
@@ -2,23 +2,23 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "joshuto";
-  version = "0.9.3";
+  version = "v0.9.4";
```
Comment (figsoda) [inline pkgs/applications/file-managers/joshuto/default.nix:5]:
```suggestion
  version = "0.9.4";
```

File: pkgs/applications/file-managers/joshuto/default.nix [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
 
   src = fetchFromGitHub {
     owner = "kamiyaa";
     repo = pname;
     rev = version;
```
Comment (figsoda) [inline pkgs/applications/file-managers/joshuto/default.nix:10]:
```suggestion
    rev = "v${version}";
```

**Review by figsoda — CHANGES_REQUESTED** [review]

**Comment by totoroot** [comment]
@figsoda Thanks! I accepted the suggestions and rebased the commits

**Comment by figsoda** [comment]
my bad, my suggestion changed the hash, can you update the hash for me?

**Comment by totoroot** [comment]
> my bad, my suggestion changed the hash, can you update the hash for me?

@figsoda Done!

**Comment by totoroot** [comment]
Result of `nixpkgs-review pr 182736` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>joshuto</li>
  </ul>
</details>

**Review by figsoda — APPROVED** [review]

**Pull request merged**